sdk/log/processor.go

```diff
@@ -26,6 +26,24 @@ type Processor interface {
 	// Before modifying a Record, the implementation must use Record.Clone
 	// to create a copy that shares no state with the original.
 	OnEmit(ctx context.Context, record Record) error
+	// Enabled returns whether the Processor will process for the given context
+	// and record.
+	//
```

Comment:
I'm not sure what this would mean in the context of a call to `Enabled`. Like they should not stop an external network call if the context is canceled?

Reply:
True. This comment could just lead to confusion.